### DISCUSSION TITLE
I followed your tutorial but have an error

### DISCUSSION THREAD

**Discussion by atelyshev** [frankframework/frankframework#4710] [Q&A] [answered]
Hello,
I created test pipeline from 001 - Get started lesson.
Then I started Framework and tried to test my new pipeline
Selected configuration Then adapter left session key empty and typed TEST in message window
Here the error from log 
2023-03-28 18:23:58,313 WARN  [http-nio-80-exec-6] {} web.ExceptionHandler - Caught unhandled Exception while executing FF!API call
com.fasterxml.jackson.databind.exc.InvalidDefinitionException: No serializer found for class java.io.ByteArrayInputStream and no properties discovered to create BeanSerializer (to avoid exception, disable SerializationFeature.FAIL_ON_EMPTY_BEANS) (through reference chain: java.util.HashMap["result"])
	at com.fasterxml.jackson.databind.exc.InvalidDefinitionException.from(InvalidDefinitionException.java:77) ~[jackson-databind-2.14.2.jar:2.14.2]

Here is my config
<Configuration name="myfirstconfig">
    <Adapter name="exampleadapter">

        <Receiver name="examplereceiver">
            <JavaListener name="examplelistener" />
        </Receiver>

        <Pipeline>
            <FixedResultPipe name="thefirstpipe" returnString="Hello Aleksey!">
                <Forward name="success" path="EXIT" />
            </FixedResultPipe>
            <Exit name="EXIT" state="success" />
        </Pipeline>

    </Adapter>
</Configuration>

And build.xml
<project default="restart-myfirstfrank">
    <target name="restart-myfirstfrank">
        <basename property="project.dir" file="${basedir}"/>
        <condition property="exe" value="../frank-runner/restart.bat" else="/bin/bash">
            <os family="windows"/>
        </condition>
        <condition property="arg" value="../frank-runner/restart.sh" else="">
            <os family="unix"/>
        </condition>
        <exec executable="${exe}" vmlauncher="false" failonerror="true">
            <arg value="${arg}"/>
            <arg value="-Dproject.dir=${project.dir}"/>
        </exec>
    </target>
</project>

Any idea what is wrong ?
Thanks
Aleksey

**Comment by nielsm5** [comment, marked as answer]
Hi!

Apologies for the delay. There were some issues with our transport layer, however they should have been fixed now.
Your safest bet is to complete the tutorial using our latest release version [7.8](https://github.com/ibissource/iaf/releases/tag/v7.8).
